src/tree/contexts/ContextTreeItem.ts

```diff
@@ -60,6 +61,18 @@ export class ContextTreeItem extends AzExtTreeItem {
     }
 
     public async use(context: IActionContext): Promise<void> {
-        return dockerContextManager.use(this.name);
+        try {
+            ext.isContextSwitchInProgress = true;
+            return this.runWithTemporaryDescription(
+                localize('vscode-docker.tree.context.using', 'Using...'),
```

Comment:
I think "changing" or "loading" is maybe a better word, even though the Docker CLI calls it "use".

Reply:
Oh, I see, this shows next to the context name in the tree. Disregard my previous suggestion, but I still think @bwateratmsft is right. "Using" suggests that the context is already being used, which is not the case here. "Changing" or "activating" conveys the state much better.

Reply:
Since the loading icon is shown next to it and also the "...," next to using, it conveys that the using is in progress. Lets see what @BigMorty thinks.

Reply:
Where on the Context node does this show, after the name instead of the description?
Regardless, I think "switching" or "loading" makes sense, would probably go with "switching".

Reply:
Now using 'Switching...'